liquibase-couchbase/src/main/java/liquibase/ext/couchbase/change/CreateBucketChange.java

```diff
@@ -71,4 +72,12 @@ public SqlStatement[] generateStatements() {
         };
     }
 
+    @Override
+    protected Change[] createInverses() {
+        DropBucketChange inverse = new DropBucketChange();
+        inverse.setBucketName(bucketName);
```

Comment:
Just a little note: may be we can use constructor with arguments here instead of calling setters(but I'm ok with this implementation as well)?Same for the other change classes, and may be to add a builder for classes with more than 3 arguments(DropIndexChange)?

Reply:
Changed to builders. Thank you